File: java/task6.java
import java.util.Scanner;
public class task6
{
    public static void main(String args[])
    {
        System.out.print("Enter a number:");
        Scanner sc = new Scanner(System.in);
        float num = sc.nextFloat();  
        if(num > 0)
            System.out.println(" positive");
        else if(num < 0)
            System.out.println(" negative");
        else
            System.out.println(" zero");
    }
}

File: c/task6.c
#include<stdio.h>
main()
{int n,t,sum=0,d;
printf("Enter the number : ");
scanf("%d",&n);
t=n;
while(n>0)
{
d=n%10;
sum= sum +d*d*d;
n=n/10;
}
if(sum==t)
printf("armstrong no.");
else
printf("not armstrong no.");
}
